dvc/command/experiments.py

```diff
@@ -1548,6 +1592,13 @@ def add_parser(subparsers, parent_parser):
     experiments_init_parser.add_argument(
         "--live", help="Path to log dvclive outputs for your experiments"
     )
+    experiments_init_parser.add_argument(
+        "--with",
```

Comment:
Not so sure about this flag, can change to anything better.

Reply:
I prefer `--type`, `--stage` has a different meaning. Compared to `--format`, `--type` is more simpler.